### DISCUSSION TITLE
Welcome to contentbox Discussions!

### DISCUSSION THREAD

**Discussion by FrancescoXX** [FrancescoXX/contentbox#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by priyankarpal** [comment]
@FrancescoXX please add issue labels it will help to check issue you can check the site https://opensource.creativecommons.org/contributing-code/repo-labels/

also, we need to follow the conventional commits check the site https://www.conventionalcommits.org/en/v1.0.0/

sometimes we add few commits like this `update the page` we need to avoid

**Reply by FrancescoXX** [reply]
I think now some label are available

**Comment by FrancescoXX** [comment]
let's go step by step an working on having a decent version 1 first

**Comment by helenhill2326** [comment]
hey

**Reply by vikasz1** [reply]
Hi

**Reply by priyankarpal** [reply]
please ask your doubts.
